packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -36,10 +37,13 @@ export class CXLMarketingNavElement extends LitElement {
   @property({ type: Boolean, reflect: true })
   minimal = false;
 
-  // vaadin-device-detector.
+  // Device Detector property.
   @property({ type: Boolean, reflect: true })
   wide;
 
+  // Device Detector media query.
+  _wideMediaQuery = '(min-width: 750px)';
```

Comment:
This seems like needs to become a cxl-lumo-styles global property?

Unless a solid immediate idea pops up, can also put it into a new PR.

Reply:
> This seems like needs to become a cxl-lumo-styles global property?
> 
> Unless a solid immediate idea pops up, can also put it into a new PR.

Yes can be dealt with in new PR with shared media query mixin or such.

Reply:
@lkraav What do you mean by global property? If decide to go the mixin route, the definition could reside there.

Reply:
> If decide to go the mixin route, the definition could reside there.

Yep, exactly that.